azavea/district-builder-dtl-pa
# Style fixes

## Overview

Fixes a few UI and styling glitches.

### Demo

#### Before

Button at bottom of the page would get covered by the black "Map name" bar: 
![image](https://user-images.githubusercontent.com/1809908/44676751-2b6cb080-aa02-11e8-8caf-349ee1ecf395.png)

#### After
![image](https://user-images.githubusercontent.com/1809908/44676783-43443480-aa02-11e8-92ef-0e0ca8ec5b48.png)

#### Before

On Leaderboards review, when "My Ranked Maps" was selected, there would be no margin between the two tabs.
![image](https://user-images.githubusercontent.com/1809908/44676865-68d13e00-aa02-11e8-927f-023b681222a0.png)

#### After

![image](https://user-images.githubusercontent.com/1809908/44676913-8c948400-aa02-11e8-84bd-a8d4953cb109.png)

#### Before

On Evaluate view, buttons would not be vertically aligned. To see the "Open Current REport in New Window" button you'll need to first click "Create and Preview New Report".

![image](https://user-images.githubusercontent.com/1809908/44676978-be0d4f80-aa02-11e8-9e41-1f39d79d1257.png)

#### After

![image](https://user-images.githubusercontent.com/1809908/44676989-c8c7e480-aa02-11e8-9687-9959b27410b4.png)

#### Before
On Evaluate view, after a report has been generated, Prev and Next buttons had inconsistent formatting.
![image](https://user-images.githubusercontent.com/1809908/44677048-f6ad2900-aa02-11e8-95f0-e3442f009fd7.png)

#### After
![image](https://user-images.githubusercontent.com/1809908/44677200-57d4fc80-aa03-11e8-8dcc-8232ce1adc70.png)

#### Before
Target Population takes up two rows on map view:
![image](https://user-images.githubusercontent.com/1809908/44677279-910d6c80-aa03-11e8-9014-bb84d044fe27.png)

#### After
![image](https://user-images.githubusercontent.com/1809908/44677289-9965a780-aa03-11e8-8cda-24caf67f0726.png)

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

 * Run application
 * Look at the Before and After screenshots in the Demo. Make sure the application looks like the After screenshot and has addressed the issue mentioned in the Before section
